dbms/src/Flash/Pipeline/Schedule/TaskQueues/TaskQueueType.h

```diff
@@ -22,5 +22,7 @@ enum class TaskQueueType
     DEFAULT, // Determined internally by the task thread pool.
     MLFQ, // multi-level feedback queue
     IO_PRIORITY, // io priority queue
+    RCQ_MLFQ, // resource control queue nesting MLFQ.
+    RCQ_IO_PRIORITY, // resource control queue nesting IO_PRIORITY.
```

Comment:
It seems that IOThreadPool should not use ResourceGroupQueue, because TiDB's resource control does not currently include io, and the execution in IOThreadPool basically does not consume cpu